schema/traces/traces.mapping

```diff
@@ -0,0 +1,220 @@
+          }
+        },
+        {
+          "links_attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "links.attributes.*"
+          }
+        },
+        {
+          "instrumentation_scope_attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "instrumentationScope.attributes.*"
+          }
+        },
+        {
+          "resources_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "resource.*"
+          }
+        }
+      ],
+      "_source": {
+        "enabled": true
+      },
+      "properties": {
+        "traceId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "spanId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "traceState": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        },
+        "parentSpanId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "name": {
+          "ignore_above": 1024,
+          "type": "keyword"
+        },
+        "kind": {
+          "ignore_above": 128,
+          "type": "keyword"
+        },
+        "startTime": {
+          "type": "date_nanos"
+        },
+        "endTime": {
+          "type": "date_nanos"
+        },
+        "droppedAttributesCount": {
+          "type": "long"
+        },
+        "droppedEventsCount": {
+          "type": "long"
+        },
+        "droppedLinksCount": {
+          "type": "long"
+        },
+        "status": {
+          "properties": {
+            "code": {
+              "ignore_above": 128,
+              "type": "keyword"
+            },
+            "message": {
+              "ignore_above": 128,
+              "type": "keyword"
+            }
+          }
+        },
+        "attributes": {
+          "type": "object",
+          "properties": {
+            "data_stream": {
+              "properties": {
+                "dataset": {
+                  "ignore_above": 128,
+                  "type": "keyword"
+                },
+                "namespace": {
+                  "ignore_above": 128,
+                  "type": "keyword"
+                },
+                "type": {
+                  "ignore_above": 56,
+                  "type": "keyword"
+                }
+              }
+            }
+          }
+        },
+        "events": {
+          "type": "nested",
+          "properties": {
+            "name": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "@timestamp": {
+              "type": "date_nanos"
+            },
+            "observedTimestamp": {
+              "type": "date_nanos"
```

Comment:
@YANG-DB  could you clarify difference between `@timestamp` and `observedTimestamp`?

I understand it as "`@timestamp` is value provided by opentelemetry" vs "`observedTimestamp` is the time exporter read the event record"

Reply:
yes - I'll add this to the README...